prac_05/emails.py

```diff
@@ -1,6 +1,7 @@
 def main():
     email_to_name = {}
     email = input("Email: ")
+
     while email != "":
         name = get_name_from_email(email)
         confirm_name = input(f"Is your name {name} (Y/n) ")
```

Comment:
Not sure Lindsay is too fussed about it but I have noticed a preference towards using the .format() method for string formatting